clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/processor/JustificationProcessor.java

```diff
@@ -121,6 +127,29 @@ public AuditLog.Builder auditLogBuilderWithJustification(
           Value.newBuilder().setStructValue(justificationStruct).build());
       auditLogBuilderCopy.setMetadata(metadataBuilder.build());
 
+      Claim justificationsClaim = jwt.getClaim("justs");
```

Comment:
better to have a constant for this string

Reply:
ack.